### DISCUSSION TITLE
Dyld code injection

### DISCUSSION THREAD

**Discussion by LydGol90** [talsec/Free-RASP-Community#29] [Q&A]
Hi there, does freeRASP protect against this kind of security attack on iOS:

```An attacker can inject dylibs into applications that modify the behavior of the application or inject arbitrary code during runtime by loading malicious dylib by abusing DYLD_INSERT_LIBRARIES env variable```

Many thanks

**Comment by syakymchuk** [comment]
hi @LydGol90, Thank you for the question. We will look at this.

**Comment by msikyna** [comment]
https://github.com/talsec/Free-RASP-Community/issues/45
